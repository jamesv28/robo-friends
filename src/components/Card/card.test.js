import {shallow} from 'enzyme';
import Card from './card.component';
import React from 'react';

it('should only render one component', () => {
    expect(shallow(<Card />).length).toEqual(1);
})

it('should take a look at snapshot testing', () => {
    expect(shallow(<Card />)).toMatchSnapshot();
})