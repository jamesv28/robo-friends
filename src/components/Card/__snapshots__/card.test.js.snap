// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a look at snapshot testing 1`] = `ShallowWrapper {}`;
